packages/neuron-ui/src/containers/Notification/index.tsx

```diff
@@ -101,17 +103,34 @@ export const NoticeContent = () => {
 
   const onGlobalDialogDismiss = useCallback(() => {
     dismissGlobalDialog()(dispatch)
+    setHasDismiss(true)
```

Comment:
It works in this `dismiss migration` case because `migration` notification always prompts first.

But it's kind of sloppy based on the assumption that `migration` notification always shows first.

I notice that `global notification` only works for the migration for now, maybe we can revise it by renaming the component.

Reply:
The `hasDismiss` variable is only used to judge whether the user have clicks dismiss for migrate, so how about rename `hasDismiss` to `hasDismissMigrate`?

Reply:
From the codebase, we can see `global notification` is only for `migration`, but its name implies that it is target-agonistic(no business keywords in the component name). So it may deliver other messages in the future.

However, `dismiss/dismissMigrate` is triggered 100% by `onGlobalDialogDismiss`.

Imagine that there're 2 messages on launch
1. `There's an update of Neuron`
2. `A migration is required`

When the global notification of an update is dismissed, the `hasDismissMigrate` will be set `true` unexpectedly.

Reply:
Currently, `global notification` also includes `Asset was released, view details in History.`
So, maybe I need to rename `onGlobalDialogDismiss ` to `onDismissMigrate`.

Reply:
> Currently, `global notification` also includes `Asset was released, view details in History.` So, maybe I need to rename `onGlobalDialogDismiss ` to `onDismissMigrate`.

It would be clear

Reply:
I found that maybe other global dialog needs to dismiss when clicking the mask, so I add a check whether the global dialog type is `rebuild-sync`  when calling `setHasDismissMigrate`. The commit is https://github.com/nervosnetwork/neuron/pull/2651/commits/d1c0d598a68425d6c245bf7a2a64bf4634bbcc33